packages/docs/developer-resources/contractkit/reference/classes/_contract_cache_.wrappercache.md

```diff
@@ -44,7 +44,7 @@ Provides access to all contract wrappers for celo core contracts
 
 \+ **new WrapperCache**(`kit`: [ContractKit](_kit_.contractkit.md)): *[WrapperCache](_contract_cache_.wrappercache.md)*
 
-*Defined in [packages/contractkit/src/contract-cache.ts:79](https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/src/contract-cache.ts#L79)*
+*Defined in [contractkit/src/contract-cache.ts:79](https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/src/contract-cache.ts#L79)*
```

Comment:
@yorhodes is this path change desired??

Reply:
I verified and at least the links are working fine